lib/cadet/accounts/accounts.ex

```diff
@@ -4,21 +4,21 @@ defmodule Cadet.Accounts do
   """
   use Cadet, :context
 
-  alias Comeonin.Pbkdf2
-
+  alias Cadet.Accounts.Authorization
+  alias Cadet.Accounts.Ivle
   alias Cadet.Accounts.User
   alias Cadet.Accounts.Query
-  alias Cadet.Accounts.Authorization
   alias Cadet.Accounts.Form.Registration
 
   @doc """
-  Register new User entity using E-mail and Password
-  authentication.
+  Register new User entity using Cadet.Accounts.Form.Registration
+
+  Returns {:ok, user} on success, otherwise {:error, changeset}
   """
   def register(attrs = %{}, role) do
     changeset = Registration.changeset(%Registration{}, attrs)
 
-    if changeset.valid?() do
+    if changeset.valid? do
```

Comment:
@indocomsoft The new style guide in #50 says that [zero-arity calls should end with parenthesis](https://github.com/lexmag/elixir-style-guide#zero-arity-parens), but I think predicate functions should be excluded, because their trailing `?` obviously indicates that they are predicate functions, not variables. Which do we go with? Your call.

Reply:
Well, in the first place, `changeset` is a map (https://hexdocs.pm/ecto/Ecto.Changeset.html#t:t/0), so `valid?` is just a key in the map, so there should definitely be no paranthesis here.